admin/static/js/sales_analytics/dashboard.js

```diff
@@ -387,19 +381,13 @@ var SalesAnalytics = function() {
                     userProductMap.osf.push(session['user.id']);
                     var paths = session.result;
                     var numberOfProducts = 0;
-                    var meetings, prereg, institutions;
```

Comment:
Should `meetings` and `institutions` still be instantiated here? I would expect that `prereg` should be removed from this line rather than the line itself being deleted.